### DISCUSSION TITLE
Deleting directories in /opt/homebrew/etc

### DISCUSSION THREAD

**Discussion by larsonius** [Homebrew/discussions#4393] [Getting started] [answered]
### Output of `brew config`

```shell
HOMEBREW_VERSION: 4.0.11
ORIGIN: https://github.com/Homebrew/brew
HEAD: 931327df1f30a3a5b1aa07657c0d1070e61988f9
Last commit: 9 days ago
Core tap origin: https://github.com/Homebrew/homebrew-core
Core tap HEAD: 990a48ea056a43bd00802adaa8e47bfce08efbdf
Core tap last commit: 7 weeks ago
Core tap branch: master
Core tap JSON: 09 Apr 10:33 UTC
HOMEBREW_PREFIX: /opt/homebrew
HOMEBREW_CASK_OPTS: []
HOMEBREW_MAKE_JOBS: 8
Homebrew Ruby: 2.6.10 => /System/Library/Frameworks/Ruby.framework/Versions/2.6/usr/bin/ruby
CPU: octa-core 64-bit arm_blizzard_avalanche
Clang: 14.0.3 build 1403
Git: 2.39.2 => /Library/Developer/CommandLineTools/usr/bin/git
Curl: 7.87.0 => /usr/bin/curl
macOS: 13.3.1-arm64
CLT: 14.3.0.0.1.1679647830
Xcode: N/A
Rosetta 2: false
```

### Output of `brew doctor`

```shell
Your system is ready to brew.
```

### Description of issue

Hello everyone,

I have a probably simple question for those who have been using homebrew longer than me:

Under /opt/homebrew/etc I have the following directories:

ca-certificates
openssl@1.0
openssl@1.1
openssl@3

But I don't have the corresponding applications installed at all, at least I don't find them in the output of "brew list".

Can I safely delete the above folders?

I am looking forward to your answers.

Best regards
larsonius

**Comment by carlocab** [comment, marked as answer]
If you don't have any of those formulae installed, then it is safe to delete them.

Just to explain why `brew` leaves them there: `etc` is a standard location for placing configuration files, so `brew` generally avoids touching files there to avoid destroying anything a user may have placed there for themselves.

**Reply by larsonius** [reply]
Thanks for your quick reply, @carlocab! 🙂
